setup_sources.sh

```diff
@@ -8,6 +8,7 @@ for spec in $dir/*.spec; do
   for source in $(spectool --list-files $spec | awk '{print $2}'); do
     sourcebase=$(basename "$source")
     [ -h $d/$sourcebase ] || continue
+    if [[ $sourcebase =~ ^.*registry\.npmjs\.org\.tgz$ ]]; then continue; fi
```

Comment:
You probably won't need this, as the `-h` check above will skip over non-symlinks.